collectors/monitoring_collector.go

```diff
@@ -506,3 +510,12 @@ func (c *MonitoringCollector) generateHistogramBuckets(
 	}
 	return buckets, nil
 }
+
+func keyExists(labelKeys []string, key string) bool {
+	for _, item := range labelKeys {
+		if item == key {
+			return true
```

Comment:
Let's make this log duplicate keys.

```suggestion
func (c *MonitoringCollector) keyExists(labelKeys []string, key string) bool {
	for _, item := range labelKeys {
		if item == key {
			level.Debug(c.logger).Log("msg", "Found duplicate label key", "key", key)
			return true
```

Reply:
Good idea

Reply:
Changes mage :+1: